backend/LexData/Migrations/20230707115233_Email verification.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace LexData.Migrations
+{
+    /// <inheritdoc />
+    public partial class Emailverification : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "EmailVerificationToken",
```

Comment:
hummmmmmmmmm.... just remove and add a new migration to clean this up

Reply:
😬 w00ps.